sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -211,4 +215,80 @@ trait ExpressionEvalHelper {
       plan(inputRow)).get(0, expression.dataType)
     assert(checkResult(actual, expected))
   }
+
+  def checkConsistency(dt: DataType, clazz: Class[_]): Unit = {
```

Comment:
For instance, this method's Scaladoc could explain that it tests the expression's one-argument constructor with randomized expressions of the given data type.